Add B20 Testnet

beonechains opened the pull request.

ligi reviewed (changes requested):
do not check in any vscode stuff

beonechains commented:
> do not check in any vscode stuff

Okey, Thanks Mr.

beonechains commented:
is it okay if there is a notification Merge blocked, what should i do?

ligi commented:
I will not merge as long as there is vscode stuff in there or this package lock

ligi reviewed (changes requested):
do not touch .gitignore (these things should be in your ignore - not in the one of this project.
Also no package-lock here

beonechains commented:
> do not touch .gitignore (these things should be in your ignore - not in the one of this project. Also no package-lock here

"I have removed the changes to .gitignore and deleted package-lock.json. Please review it again. Thank you!"

beonechains commented:
"I see that my changes have been approved, but merging is still blocked with the message 'Merging is blocked.' Should I choose 'Update with Merge Commit' or 'Update with Rebase,' or just leave it as is? Please advise. Thank you!"

ligi commented:
argh - you are wasting my time

![image](https://github.com/user-attachments/assets/144c8824-aa30-49fe-9e5a-099905745440)

beonechains commented:
hello mr. @ligi Is there anything else wrong that I did? Please correct me.

ligi reviewed (changes requested):
yes - you deleted the gitignore

beonechains commented:
> yes - you deleted the gitignore

"I have restored the .gitignore file and synced with the latest changes from master. Please review it again. Thank you mr. @ligi ! 🙏"

beonechains reviewed (commented):
I've made the requested updates, but some parts might need further clarification. Please review and let me know if any additional changes are needed. Thanks!

ligi commented:
I will not review this PR anymore without payment to lists.eth - you are wasting my time and my time is valuable.

![image](https://github.com/user-attachments/assets/9deca497-7300-4bb9-8daa-57b5ef236638)

beonechains commented:
> I will not review this PR anymore without payment to lists.eth - you are wasting my time and my time is valuable.
> 
> ![image](https://private-user-images.githubusercontent.com/111600/424371201-9deca497-7300-4bb9-8daa-57b5ef236638.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDI0NDExOTQsIm5iZiI6MTc0MjQ0MDg5NCwicGF0aCI6Ii8xMTE2MDAvNDI0MzcxMjAxLTlkZWNhNDk3LTczMDAtNGJiOS04ZGFhLTU3YjVlZjIzNjYzOC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMzIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDMyMFQwMzIxMzRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04ZjFjMDEzYjkzZGMwMmUxOWMzMmRhYjE5YTcwMzg3ZThhYmRjNmQ4MzE0OGU5ODA3YzI1MGY4MmU0ZWI0NGFkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.n41bVeURfoJu743s_LX_yWznAKhs_wWZNpDyAVwhIWo)

I have done it. Please check and I ask for your help.

ligi commented:
merged via 7062

beonechains commented:
> merged via 7062

Hi Sir, I want to ask. Why hasn't it appeared on the chainlist?
